mailpoet/.env.sample

```diff
@@ -1,10 +1,10 @@
-WP_ROOT=/var/www/html
+WP_ROOT=/wp-core
 WP_TEST_CACHE_PATH=/tmp
 WP_TEST_MAILER_ENABLE_SENDING=true
 WP_TEST_ENABLE_NETWORK_TESTS=true
 
 # Optional: for multisite acceptance tests
-WP_ROOT_MULTISITE=/var/www/wordpress
+WP_ROOT_MULTISITE=/wp-core
```

Comment:
I'm wondering what you get for `WP_ROOT`  in `tests/integration/_bootstrap.php` ? it doesn't work for me if the value is not changed to `/wp-core`, maybe it's cached in a container for you?
I can introduce new variables as well to avoid conflict

Reply:
I added a breakpoint into `_bootstrap.php` and I'm getting `/wp-core`. It seems to be because it is defined in [docker-compose.yml](https://github.com/mailpoet/mailpoet/blob/984fa678bc75d0e8e1b97e8232a3604882140bd8/mailpoet/tests/docker/docker-compose.yml#L19)
When I remove the definition of WP_ROOT from the docker-compose.yml I start getting the value from my .env